orangecontrib/timeseries/widgets/owdifference.py

```diff
@@ -17,32 +20,48 @@ class OWDifference(widget.OWWidget):
                   '1st or 2nd order discrete difference along its values. '
     icon = 'icons/Difference.svg'
     priority = 570
+    keywords = ['difference', 'derivative', 'quotient', 'percent change']
 
     class Inputs:
         time_series = Input("Time series", Table)
 
     class Outputs:
         time_series = Output("Time series", Timeseries)
 
+    class Operation(Enum):
```

Comment:
If you make this a [StrEnum](https://docs.python.org/3/library/enum.html#others), i.e. `class Operation(str, Enum)`, you won't need all those extra `Operation.*.value` calls.